### PR TITLE
Finish frontend

### PR DESCRIPTION
**Pull request by anhduy1202**
# Description
Done will frontend including /success. /error, /verify
# Screenshots
image
image
image

### CODE REVIEW

File: components/ResultBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import React from "react";
+import { useResponsive } from "../Hooks/useResponsive";
+
+type layoutPtops = {
```
Comment (jaasonw) [inline components/ResultBanner.tsx]: typo should be layoutProps?

File: components/ResultBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    <section
+      className={`flex ${!isDesktop && "flex-col"} bg-[#161b22] p-4 rounded-xl justify-center items-center md:p-8`}
+    >
+      {children}
+    </section>
+  );
+};
+
+type imageProps = {
+  source: string;
```
Comment (jaasonw) [inline components/ResultBanner.tsx]: nitpick: can we use src to be consistent with the img tag?

File: pages/error.tsx [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
-      <h1>
-        Uh oh! We weren&apos;t able to verify you, double check that you arent
-        already registered and contact an administrator and include the code
-        below if the problem persists after trying the verification link again
-      </h1>
-      <pre>{query.code}</pre>
-    </>
+    <BannerLayout>
+      <BannerImage source="/Sad Frank.svg"/>
+      <BannerContent success={false} header="Uh oh! Register Failed" />
```
Comment (jaasonw) [inline pages/error.tsx]: "Register Failed" -> "Registration Failed"

**Review by jaasonw — CHANGES_REQUESTED** [review]
look pretty good, just small nitpicks + the thing about the frank outlines

File: pages/error.tsx (hunk 3 above)
Comment (jaasonw) [reply pages/error.tsx]: also can you change the text to "Contact an administrator by clicking the link below and asking in #register-help"

File: components/ResultBanner.tsx (hunk 1 above)
Comment (anhduy1202) [reply components/ResultBanner.tsx]: done

File: components/ResultBanner.tsx (hunk 2 above)
Comment (anhduy1202) [reply components/ResultBanner.tsx]: done

File: pages/error.tsx (hunk 3 above)
Comment (anhduy1202) [reply pages/error.tsx]: done

**Review by jaasonw — APPROVED** [review]
thank you cutie 😘

**Pull request merged**